flocx_market/api/app.py

```diff
@@ -43,15 +43,16 @@ def create_app(app_name):
         OfferContractRelationship,
         '/offer_contract_relationship',
         '/offer_contract_relationship/',
-        '/offer_contract_relationship/'
-        '<string:marketplace_offer_id>/<string:contract_id>')
+        '/offer_contract_relationship'
+        '/<string:offer_contract_relationship_id>')
     api.add_resource(Root, '/')
 
     orm.init_app(app)
 
     @app.before_request
     def before_request():
         g.context = ctx.RequestContext.from_environ(request.environ)
+        # g.context.is_admin = True
```

Comment:
Ah, probably want to get rid of this : )

Reply:
:D the sign of curl testing :D